velox/functions/sparksql/DateTimeFunctions.h

```diff
@@ -227,6 +235,22 @@ struct UnixTimestampParseWithFormatFunction
     return true;
   }
 
+  FOLLY_ALWAYS_INLINE void call(
+      int64_t& result,
+      const arg_type<Timestamp>& input,
+      const arg_type<Varchar>& /*format*/) {
```

Comment:
Do we need the 'format' parameter if it is not going to be used by date and timestamp types?